aten/src/ATen/cuda/CUDAGraph.h

```diff
@@ -52,10 +52,6 @@ struct TORCH_CUDA_CPP_API CUDAGraph {
   // Set to true in capture_end if cudaGraphInstantiate succeeded
   bool has_graph_exec_ = false;
 
-  // uuid of this instance's current capture, used to
-  // specify the pool.
-  CaptureId_t id_;
```

Comment:
What happened to this?

Reply:
`id_` variable seems to be no longer used for cudaStreamGetCaptureInfo (refer to: https://github.com/pytorch/pytorch/pull/106570). Currently, it only assigns an id to a MemPool when internal data structures (like CUDAGraph) create the pool. I've moved that logic to the MemPool class, and used that instead in CUDAGraph.